sbnana/SBNAna/load_cafana_libs.C

```diff
@@ -98,7 +98,7 @@ void load_cafana_libs()
     gROOT->Macro("${SBNANA_FQ_DIR}/bin/rootlogon.C");
   }
   else{
-    gROOT->Macro("${MRB_BUILDDIR}/sbnana/bin/rootlogon.C");
+    gROOT->Macro("${SBNANA_DIR}/sbnana/CAFAna/rootlogon.C");
```

Comment:
This new version appears to mirror the setup strategy in `cafe` script.
Nevertheless I do not understand the motivation of this change. Can you explain?

Reply:
Without this PR, users ran `mrbslp` before they run `cafe`. `${SBNANA_FQ_DIR}` is only set after `mrbslp`, and there we look for "${SBNANA_FQ_DIR}/bin/rootlogon.C" which exists. This PR helps user to run `cafe` without running `mrbslp`, and there we have empty `${SBNANA_FQ_DIR}`. I then found we don't have "${MRB_BUILDDIR}/sbnana/bin/rootlogon.C", but it exists under "${SBNANA_DIR}/sbnana/CAFAna/rootlogon.C"

Reply:
While checking more on this, it looks like just using tagged `sbnana` with `setup sbnana v09_75_03 -qe20:prof` does not suffer from this issue, where most of the users (I guess) check out the product and use local products